src/manipulators/manipulator.ts

```diff
@@ -59,4 +59,45 @@ export abstract class Manipulator {
       return commentRange.getText().includes(comment);
     });
   }
+
+  /**
+   * Adds value to a Map with Set value types.
+   * @param {Map<K, Set<V>>} map - Map to add to.
+   * @param {K} key - Key to insert value at.
+   * @param {V} val - Value to insert.
+   */
+  addToMapSet<K, V>(map: Map<K, Set<V>>, key: K, val: V): void {
+    if (map.has(key)) {
```

Comment:
Ohh that's a really good point, I've never really considered that before but it makes a lot of sense. I've gone ahead and changed it to your way because it also looks cleaner too :)).